src/test/java/org/folio/circulation/domain/OverdueFineCalculatorServiceTest.java

```diff
@@ -233,54 +235,46 @@ public void shouldNotCreateFeeFineRecordWhenOwnerDoesNotExist()
     Loan loan = createLoan();
 
     when(overdueFinePolicyRepository.findOverdueFinePolicyForLoan(any()))
-      .thenReturn(CompletableFuture.completedFuture(Result.succeeded(loan)));
+      .thenReturn(completedFuture(succeeded(loan)));
     when(overduePeriodCalculatorService.getMinutes(any(), any()))
-      .thenReturn(CompletableFuture.completedFuture(Result.succeeded(5)));
+      .thenReturn(completedFuture(succeeded(5)));
     when(itemRepository.fetchItemRelatedRecords(any()))
-      .thenReturn(CompletableFuture.completedFuture(Result.succeeded(createItem())));
-    doReturn(CompletableFuture.completedFuture(Result.succeeded(null)))
-      .when(feeFineOwnerRepository).getFeeFineOwner(SERVICE_POINT_ID.toString());
+      .thenReturn(completedFuture(succeeded(createItem())));
+    when(feeFineRepository.getFeeFine(FEE_FINE_TYPE, true))
+      .thenReturn(completedFuture(succeeded(createFeeFine())));
+    when(feeFineOwnerRepository.getFeeFineOwner(SERVICE_POINT_ID.toString()))
+      .thenReturn(completedFuture(succeeded(null)));
 
     CheckInProcessRecords records = new CheckInProcessRecords(
       CheckInByBarcodeRequest.from(createCheckInByBarcodeRequest()).value())
       .withLoan(loan);
 
     overdueFineCalculatorService.calculateOverdueFine(records).get();
-    verifyNoInteractions(feeFineRepository);
     verifyNoInteractions(accountRepository);
   }
 
   @Test
-  public void shouldCreateFeeFineTypeWhenFeeFineTypeDoesNotExist()
+  public void shouldNotCreateFeeFineRecordWhenFeeFineDoesNotExist()
     throws ExecutionException, InterruptedException {
     Loan loan = createLoan();
 
     when(overdueFinePolicyRepository.findOverdueFinePolicyForLoan(any()))
-      .thenReturn(CompletableFuture.completedFuture(Result.succeeded(loan)));
+      .thenReturn(completedFuture(succeeded(loan)));
     when(overduePeriodCalculatorService.getMinutes(any(), any()))
-      .thenReturn(CompletableFuture.completedFuture(Result.succeeded(periodCalculatorResult)));
+      .thenReturn(completedFuture(succeeded(5)));
```

Comment:
Why is this value hardcoded? Could it be dependent on the test parameters?

Reply:
yes, fixed